packages/itwinui-react/src/styles.js/styles.module.css

```diff
@@ -7,3 +7,9 @@
 
 @import '@itwin/itwinui-variables' layer(itwinui.v3);
 @import '@itwin/itwinui-css' layer(itwinui.v3);
+
+@layer itwinui.v3 {
+  .iui-root {
+    --_iui-v3-loaded: yes;
```

Comment:
Any advantage of choosing `yes` instead of something like `true`?

Reply:
i deliberately didn't use "true" because this is a string and i don't want to confuse it with the Javascript value `true`